packages/cubejs-schema-compiler/src/adapter/PreAggregations.js

```diff
@@ -107,8 +112,8 @@ export class PreAggregations {
 
   preAggregationDescriptionFor(cube, foundPreAggregation) {
     const { preAggregationName, preAggregation } = foundPreAggregation;
-    const name = foundPreAggregation.sqlAlias || preAggregationName;
```

Comment:
`foundPreAggregation ` doesn't contain `sqlAlias`. `foundPreAggregation. preAggregation` should be used.